wurst/systems/modes/GameModeInit.wurst

```diff
@@ -0,0 +1,335 @@
+public constant string array TEST_MODE              = ["test-mode"             , "tm"   , "No one can lose, enables test commands"]
+public constant string array START                  = ["start"                 , "st"   , "Start the game by entering class selection"]
+public constant string array SAFE                   = ["safe"                  , "sa"   , "Stats degrade slower every 4.5 s instead of 3 s"]
+public constant string array INCREASE_PETS          = ["increase-pets"         , "ip"   , "Pets spawn 23% of the time instead of 17%"]
+public constant string array RAISE_PET_GROWTH       = ["raise-pet-growth"      , "rpg"  , "Increase pet growth chance to 30%"]
+public constant string array LAVISH                 = ["lavish"                , "lv"   , "More animals spawn and they give more food"]
+public constant string array RAISE_FOOD_LIMIT       = ["raise-food-limit"      , "rfl"  , "More animals and fish can be on the map at the same time"]
+public constant string array FAST                   = ["fast"                  , "fs"   , "More items spawn at once"]
+public constant string array RAISE_ITEM_LIMIT       = ["raise-item-limit"      , "ril"  , "Increase item spawn limits"]
+public constant string array NO_INVENTORY_LIMIT     = ["no-inventory-limit"    , "nil"  , "Remove inventory limits on axes/spells/armor"]
+public constant string array SURVIVAL               = ["survival"              , "sv"   , "This mode combines famine, cold, and slow mode"]
+public constant string array PANIC                  = ["panic"                 , "pn"   , "Stats degrade faster at 2.3 s vs 3 s"]
+public constant string array DOUBLE                 = ["double"                , "db"   , "Stats degrade twice as fast"]
+public constant string array FAMINE                 = ["famine"                , "fm"   , "Animals spawn less"]
+public constant string array FAMINE_X2              = ["famine-x2"             , "fm2"  , "More intense famine"]
+public constant string array SLOW                   = ["slow"                  , "sl"   , "Spawn less items"]
+public constant string array LOWER_FOOD_LIMIT       = ["lower-food-limit"      , "lfl"  , "Lower spawn rate for animals on the map"]
+public constant string array LOWER_ITEM_LIMIT       = ["lower-item-limit"      , "lil"  , "Lower item spawn rate on the map"]
+public constant string array OLD_ELEMENTALIST       = ["old-elementalist"      , "oe"   , "Replaces new elementalist with the old elementalist."]
+public constant string array DEV                    = ["dev"                   , "d"    , "Development mode (-qp, -tm, -st)"]
+public constant string array TAMEABLE_FAWN          = ["tameable-fawn"         , "tf"   , "Tame fawns with acorns"]
+
+
+init
+    GameMode.initialize()
+
+    new GameMode(PLAYER_PER_TRIBE[0], PLAYER_PER_TRIBE[1], PLAYER_PER_TRIBE[2]) (mode, args) ->
```

Comment:
Can you not avoid unpacking each array? Could you make the `GameMode` constructor take in an array of strings instead?